source/CMakeLists.txt

```diff
@@ -414,7 +414,9 @@ if("${CMAKE_SYSTEM_NAME}" STREQUAL "Linux")
   find_library(LIBRT rt)
   if(LIBRT)
     foreach(target ${SPIRV_TOOLS_TARGETS})
-      target_link_libraries(${target} rt)
+      target_link_libraries(${target} PRIVATE rt)
+      # spirv-dis's --nested-indent uses helpers from spirv-opt.
+      target_link_libraries(${target} PRIVATE SPIRV-Tools-opt)
```

Comment:
This is too much code to add to the core library.    Also the optimizer lib depends on the core lib, so it's circular (which is bad).

Reply:
Fixed